homeassistant/components/lamarzocco/switch.py

```diff
@@ -0,0 +1,193 @@
+"""Switch platform for La Marzocco espresso machines."""
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+from typing import Any
+
+from lmcloud.const import LaMarzoccoModel
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import LaMarzoccoEntity, LaMarzoccoEntityDescription
+from .lm_client import LaMarzoccoClient
+
+ATTR_MAP_MAIN_GS3_AV = [
+    "dose_k1",
+    "dose_k2",
+    "dose_k3",
+    "dose_k4",
+    "dose_k5",
+]
+
+ATTR_MAP_AUTO_ON_OFF = [
+    "mon_auto",
+    "mon_on_time",
+    "mon_off_time",
+    "tue_auto",
+    "tue_on_time",
+    "tue_off_time",
+    "wed_auto",
+    "wed_on_time",
+    "wed_off_time",
+    "thu_auto",
+    "thu_on_time",
+    "thu_off_time",
+    "fri_auto",
+    "fri_on_time",
+    "fri_off_time",
+    "sat_auto",
+    "sat_on_time",
+    "sat_off_time",
+    "sun_auto",
+    "sun_on_time",
+    "sun_off_time",
+]
+
+ATTR_MAP_PREBREW_GS3_AV = [
+    "prebrewing_ton_k1",
```

Comment:
Please reduce extra state attributes to a minimum

See https://developers.home-assistant.io/docs/core/entity
> Entities that generate a significant amount of state changes can quickly increase the size of the database when the extra_state_attributes also change frequently. Minimize the number of extra_state_attributes for these entities by removing non-critical attributes or creating additional sensor entities.

What are the meanings of all that `k`'s?

Reply:
state changes are minimal. I removed all the extra attributes for now, we can discuss about that in a later PR 😉